src/QAT/qat/purr/backends/live_devices.py

```diff
@@ -74,10 +74,11 @@ class ControlHardwareChannel(PhysicalChannel):
     derived) object should contain hardware specific information.
     """
 
-    def __init__(self, id_, hardware_id, dcbiaschannel_pair, *args, **kwargs):
+    def __init__(self, id_, hardware_id, dcbiaschannel_pair, *args, switch_ch=None, **kwargs):
```

Comment:
this should be before *args

Reply:
It's a `kwarg`, why would it be before `*args`?

Reply:
ah you're right, sorry! Misread with test wrapping